src/zeek-setup.cc

```diff
@@ -582,7 +582,7 @@ SetupResult setup(int argc, char** argv, Options* zopts) {
 
     if ( options.supervisor_mode ) {
         Supervisor::Config cfg = {};
-        cfg.zeek_exe_path = zeek_exe_path;
+        cfg.zeek_exe_path = std::move(zeek_exe_path);
```

Comment:
I guess I understand where this comes from, but if someone started to use `zeek_exe_path` further down in this very long function, this has great bug/confusion potential (this applies to other places as well). Performance here is irrelevant, so much I'd almost suggest another call to `util::detail::get_exe_path(argv[0])` rather than this `std::move()`.

For pleasing coverity sure, but it's very noisy and I can't say for the better (but then my limit for surprises is also pretty low).

Reply:
Yeah, this one isn't going to make or break any sort of performance. I'll drop it and mark the Coverity issue as ignored.

Reply:
@vpax echoed both of your concerns here as well: (a) there's plenty of places where this could trip someone up making later changes where a value as already been `move`d and (b) these add a lot of visual noise.